apps/evm/src/config/rpcUrls.ts

```diff
@@ -6,6 +6,7 @@ export const rpcUrls: {
   [chainId in ChainId]: string[];
 } = {
   [ChainId.BSC_MAINNET]: [
+    `https://bnb-mainnet.g.alchemy.com/v2/${envVariables.VITE_ALCHEMY_API_KEY}`,
```

Comment:
Alchemy should be set as the second available option, otherwise it will be used as the main RPC provider